website/docs/r/application_gateway.html.markdown

```diff
@@ -302,6 +302,8 @@ A `http_listener` block supports the following:
 
 * `custom_error_configuration` - (Optional) One or more `custom_error_configuration` blocks as defined below.
 
+* `firewall_policy_id` - (Optional) The resource ID of a firewall policy.
```

Comment:
we intentionally avoid the phrase `Resource ID` since it's ambiguous between Terraform and Azure Resource ID's - which aren't always the same - as such could we make this:

```suggestion
* `firewall_policy_id` - (Optional) The ID of the Web Application Firewall Policy which should be used as a HTTP Listener.
```

(also worth clarifying this is a "Web Application" Firewall Policy, since Azure Firewall reuses the name too)

Reply:
fixed it in next commit. Found another entry (not on http listener level but at highest level of an app gw) as I copied it from there